libraries/AP_Mount/AP_Mount_Xacti.cpp

```diff
@@ -43,11 +36,20 @@ AP_Mount_Xacti::AP_Mount_Xacti(class AP_Mount &frontend, class AP_Mount_Params &
     param_int_cb = FUNCTOR_BIND_MEMBER(&AP_Mount_Xacti::handle_param_get_set_response_int, bool, AP_DroneCAN*, const uint8_t, const char*, int32_t &);
     param_string_cb = FUNCTOR_BIND_MEMBER(&AP_Mount_Xacti::handle_param_get_set_response_string, bool, AP_DroneCAN*, const uint8_t, const char*, AP_DroneCAN::string &);
     param_save_cb = FUNCTOR_BIND_MEMBER(&AP_Mount_Xacti::handle_param_save_response, void, AP_DroneCAN*, const uint8_t, bool);
+
+    // static assert that Param enum matches parameter names array
+    static_assert((uint8_t)AP_Mount_Xacti::Param::LAST+1 == ARRAY_SIZE(AP_Mount_Xacti::_param_names), "AP_Mount_Xacti::_param_names array must match Param enum");
 }
 
 // init - performs any required initialisation for this instance
 void AP_Mount_Xacti::init()
 {
+    // instantiate parameter queue, return on failure so init fails
+    _set_param_int32_queue = new ObjectArray<SetParamQueueItem>(XACTI_SET_PARAM_QUEUE_SIZE);
+    if (_set_param_int32_queue == nullptr) {
+        gcs().send_text(MAV_SEVERITY_ERROR, "%s init failed", send_text_prefix);
```

Comment:
```suggestion
        GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "%s init failed", send_text_prefix);
```

..everywhere and always, I think.

Reply:
Txs, replaced gcs().send_text() with GCS_SEND_TEXT() everywhere in the .cpp.